### ISSUE TITLE
FilteredFormat: Since latest Chrome / Firefox Dev: Uncaught TypeError: fixedValues.split is not a function

### ISSUE THREAD

**Issue by Fannon** [SemanticMediaWiki/SemanticResultFormats#158]
Hello,

I've noticed this bug in FilteredFormats for a while happening in Chrome Dev and Firefox Dev browsers. The change leading to this bug seems to have recently landed in Chrome stable, as my main browser is having the same issue now, too.

Can someone else reproduce this, too?

## Steps to reproduce:
* Use a very recent browser (chrome dev, chrome latest, firefox nightly)
* Go to a page using the filtered-format result format with some filters active
* The filters are not displayed, as the JavaScript error stops the execution of the result formats script

![2016-06-13 12 24 44](https://cloud.githubusercontent.com/assets/470980/16004454/0697f974-3162-11e6-8271-bd9f004da2cb.png)

```
ext.srf.filtered.value-filter.js:157 Uncaught TypeError: fixedValues.split is not a function
init	@	ext.srf.filtered.value-filter.js:157
valueFilter	@	ext.srf.filtered.value-filter.js:305
(anonymous function)	@	ext.srf.filtered.js:117
each	@	load.php?debug=true&lang=en&modules=jquery%2Cmediawiki&only=scripts&skin=chameleon&version=20160610…:384
each	@	load.php?debug=true&lang=en&modules=jquery%2Cmediawiki&only=scripts&skin=chameleon&version=20160610…:136
attachFilter	@	ext.srf.filtered.js:103
$.fn.filtered	@	ext.srf.filtered.js:347
(anonymous function)	@	ext.srf.filtered.value-filter.js:317
(anonymous function)	@	ext.srf.filtered.value-filter.js:319
```

**Comment by kghbln** [comment]
@mitchelln confirmed that C 51 starts to have the problem. Ping @s7eph4n as he was the original author of this cool format.

**Comment by mitchelln** [comment]
Oh dear, this will be an absolute disaster. I have many sites running the filtered result format.

Happy to sponsor a fix for this.

**Comment by Fannon** [comment]
I can confirm that this is fixed with pull request #160

Thanks, @s7eph4n !

**Comment by kghbln** [comment]
So this will be in [SRF 2.4](https://github.com/SemanticMediaWiki/SemanticResultFormats/milestones/SRF%202.4)
